src/adapters/netsuite/index.js

```diff
@@ -62,7 +62,8 @@ async function findContact({ user, authHeader, phoneNumber, overridingFormat })
     const foundContacts = [];
```

Comment:
Here we'll need the use of overridingFormat, to support non-E.164 formats like: (123) 456-7890. The way to do it is, the setup is in extension options which allows up to 3 formats in very simple matter - replacing all * signs one by one with raw number (if you get number +11234567890, then raw number is 1234567890, refer to Clio for further details.)

Reply:
Addressed